### ISSUE TITLE
styles for rows or individual cells

### ISSUE THREAD

**Issue by tyleraadams** [MerlinLabs/styled-material-components#268]
thanks to the creators/maintainers of this awesome project. I am seeing some really great implementations of material ui with styled components here. 

One small issue I am having with the table component is I'd like to be able to apply classNames to individual cells or rows so that I can manipulate styles. I see you have an example for how to add custom styles on a column basis, but any plans to apply this on a more granular level?

**Comment by martahj** [comment]
We don't have a way of supporting that right now, but might consider doing so in the future (and would happily accept a PR). A way to achieve a similar effect would be to pass a react component with the desired class to individual cells. For example, if you had a table of price events like in the [demo docs](https://github.com/MerlinLabs/styled-material-components/blob/master/pages/tables.js) and you wanted some of the cells in the `price` column to be different colors, you could do something like:

```js
const data = [
  {
    key: 'ssssdas',
    date: '02/01/17',
    event: 'Showing',
    price: '$14,500,000',
  },
  {
    key: '3oirj923o',
    date: '02/01/17',
    event: 'Price Change',
    price: <div className="yellow-text">{'$24,500,000'}</div>
  },
  {
    key: 'xzvxzcv',
    date: '02/01/17',
    event: 'Open House',
    price: <div className="blue-text">'{$5,500,000'}</div>
  },
];
```

Would something like that work for your use case?

**Comment by tyleraadams** [comment]
Yes that works perfectly for my use case, thanks! the solution I forked just passes along a className prop to the row object, and that passes it down to the Datum component.
